includes/gateway/class-omise-payment-alipay.php

```diff
@@ -61,102 +61,60 @@ public function init_form_fields() {
 		}
 
 		/**
-		 * @param  int $order_id
+		 * @since  3.3
 		 *
-		 * @see    WC_Payment_Gateway::process_payment( $order_id )
-		 * @see    woocommerce/includes/abstracts/abstract-wc-payment-gateway.php
+		 * @see    Omise_Payment::process_payment( $order_id )
 		 *
-		 * @return array
+	 	 * @param  int $order_id
+		 * @param  WC_Order $order
+		 *
+		 * @return OmiseCharge|OmiseException
 		 */
-		public function process_payment( $order_id ) {
```

Comment:
Note: This change might a bit hard to see because of GitHub trying to differ between the old code and a new one but basically, 
1. `process_payment` will be renaming to `charge`.

Then removing all `load order` and `add note to order` logic (moving it out to `Omise_Payment::process_payment`).